solidity_contracts/src/CairoPrecompiles/DualVmToken.sol

```diff
@@ -285,7 +286,8 @@ contract DualVmToken is NoDelegateCall {
         transferCallData[1] = uint256(amountLow);
         transferCallData[2] = uint256(amountHigh);
 
-        starknetToken.delegatecallCairo("transfer", transferCallData);
+        bool success = abi.decode(starknetToken.delegatecallCairo("transfer", transferCallData), (bool));
+        require(success, "Transfer failed");
```

Comment:
use solidity errors instead of require statement to keep the same way of handling errors